salesforce/latest/using/community.md

```diff
@@ -136,4 +136,39 @@ You will see a **Salesforce Community Member Access** message. The result of the
 
       ![lightning-alfresco-tab]({% link salesforce/images/lightning-alfresco-tab.png %})
 
-  Community Members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.
\ No newline at end of file
+  Community Members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.
```

Comment:
This is not limited to cases.  Any Salesforce sObject or custom object that is used in a community can be used with the connector.

Reply:
This  line  is already present for previous section. Its not added by me.